api.md

```diff
@@ -891,14 +891,14 @@ runs a reorder on the `_timescaledb_internal._hyper_1_10_chunk` chunk using the
 ## Continuous Aggregates :community_function: [](continuous-aggregates)
 TimescaleDB allows users the ability to automatically recompute aggregates
 at predefined intervals and materialize the results. This is suitable for
-frequently used queries.
+frequently used queries. For a more detailed discussion of them see [using TimescaleDB Continuous Aggregates][using-continuous-aggs]
```

Comment:
"For a more detailed discussion of this capability, please see [using TimescaleDB Continuous Aggregates][using-continuous-aggs]."

(Note also period at end of sentence)